src/style.css

```diff
@@ -1 +1,58 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
```

Comment:
for better performance, it is recommended to import fonts in your HTML using the `link` tag.